geniza/corpus/migrations/0020_populate_documenttype_display_label.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 3.2.8 on 2021-10-25 13:41
+
+from django.db import migrations
+
+
+def populate_document_types(apps, schema_editor):
+    DocumentType = apps.get_model("corpus", "DocumentType")
+    DocumentType.objects.update_or_create(
+        name="Legal", defaults={"display_label": "Legal document"}
+    )
+    DocumentType.objects.update_or_create(
+        name="Letter", defaults={"display_label": "Letter"}
+    )
+    DocumentType.objects.update_or_create(
+        name="List or table", defaults={"display_label": "List or table"}
+    )
```

Comment:
one more: there's no need to set the display label when it is the same as the name, so please drop these two